textBasedAdventure.js

```diff
@@ -1,5 +1,190 @@
 const readline = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readline.question("Please enter your name: ")
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+let name = nameInput
+const fail = "you died"
+const beat = "Congrats you beat have escaped"
+const secret = "you've found the super mario secret pipe that sends you to world 3, you beat the game. You and Ban have escaped congrats... tell no one about this."
+const theTen = Math.random
+const arr2 = [1,2,3,4,5,6,7,8,9]
+const arr = ["blue","yellow","red","green","black","white","brown","pink","silver"]
+
+
+const roomThree = () => {
+    console.log(" ")
+    userInput = readline.question("Congratulations, \
+    You and Ban enter Room 3" + " \
+    Room 3 contains six doors\
+     " + "make your pick " + "\
+     please enter a number between 1 and 6 ")
+        if(userInput === "1" ||userInput === "2"||userInput === "3") {
+            roomFour()
+        } else {    
+            console.log(fail)
+        }
+    }
+
+    const roomFour = () => {
```

Comment:
All the rooms have the same scope. RoomThree and forth could have the same tab indentation, you only need to go deeper in the tab indentation if you have are going deeper in scope. 
like this.
`const test = (arr)=> {
        for (let el of arr){
               if(el<20){
                    console.log('End of test')
               }
        }
 }
`